kalite/control_panel/api_resources.py

```diff
@@ -402,7 +407,7 @@ def alter_list_data_to_serialize(self, request, to_be_serialized):
             bundle.data.pop("user")
 
             content_id = bundle.data.pop("content_id", None)
-            content = get_content_data(request, content_id) or get_exercise_data(request, content_id)
+            content = get_content_item(language=request.language, content_id=content_id)
```

Comment:
This is probably OK - but we have the slight issue of duplicate content_ids in the topic tree. (Note: this is an issue both with content items being repeated in the topic tree, and also with Topics and exercises sometimes having the same 'id').

We should switch to UUIDs.